Drop after graphics/cairo >= 1.18 update

Dependency cairo found: NO. Found 1.17.4 but need: '>= 1.18.0'
Did not find CMake 'cmake'
Found CMake: NO
Run-time dependency cairo found: NO (tried pkgconfig and cmake)
Looking for a fallback subproject for the dependency cairo

meson.build:395:17: ERROR: Git program not found, cannot download cairo.wrap via git.

--- meson.build.orig	2024-09-13 14:11:36 UTC
+++ meson.build
@@ -16,7 +16,7 @@ fribidi_req        = '>= 1.0.6'
 pango_req          = '>= 1.52.0' # keep this in sync with .gitlab-ci/test-msys.sh
 harfbuzz_req       = '>= 2.6.0'
 fribidi_req        = '>= 1.0.6'
-cairo_req          = '>= 1.18.0'
+cairo_req          = '>= 1.17.0'
 gdk_pixbuf_req     = '>= 2.30.0'
 wayland_proto_req  = '>= 1.36'
 wayland_req        = '>= 1.21.0'
